commands/debug.js

```diff
@@ -65,8 +65,15 @@ if(args[0].toLowerCase()==="quest"){
 
 var messagePing = message.mentions.members.first();
 var targetID = messagePing ? message.guild.id.concat(messagePing.id) : userid;
-charid = client.userMap.get(targetID,"possess");
-sburbid = client.charcall.allData(client,targetID,charid,"owner");
+if(client.userMap.has(targetID))
+{
+	charid = client.userMap.get(targetID,"possess");
```

Comment:
Small problem with this line, if a person has sent a message (like a DM) but hasn't registered yet, they will have userData but their possess will be listed as "NONE", which this will store as the charid, so we need one more check to see that charid isn't "NONE" before proceeding to the rest of the code.